app/controllers/InstitutionSelectAddressController.scala

```diff
@@ -48,26 +52,57 @@ class InstitutionSelectAddressController @Inject() (
 
   def onPageLoad(mode: Mode): Action[AnyContent] = (identify andThen getData andThen requireData) {
     implicit request =>
-      val preparedForm = request.userAnswers.get(InstitutionSelectAddressPage) match {
-        case None        => form
-        case Some(value) => form.fill(value)
-      }
+      request.userAnswers.get(AddressLookupPage) match {
+        case Some(addresses) =>
+          val preparedForm: Form[String] = request.userAnswers.get(InstitutionSelectAddressPage) match {
+            case None        => form
+            case Some(value) => form.fill(value)
+          }
+
+          val radios: Seq[RadioItem] = addresses.map(
```

Comment:
can we use a more descriptive name for 'radios'? These are the returned address from the lookup service right?

Reply:
Change name to radioAddressFields